subjects/devops/orchestrator/audit/README.md

```diff
@@ -0,0 +1,232 @@
+
+- Node.JS applications must be deployed as a deployment and they must be scaled horizontally automatically, depending on CPU consumption:
+
+1. `api-gateway`:
+   max replication: 3
+   min replication: 1
+   cpu percent triger: 60%
+
+2. `billing-app`:
+   max replication: 3
+   min replication: 1
+   cpu percent triger: 60%
+
+3. `inventory-app`:
+   max replication: 3
+   min replication: 1
+   cpu percent triger: 60%
+
+###### Do all apps deploy with the correct configuration?
+
+##### Ask the following questions to the group or student
+
+- What is statefulset in k8s?
+
+- What is deployment in k8s?
+
+- What is the difference between deployment and statefulset in k8s?
+
+- What is scaling and why do we use it?
+
+- What is a load balancer and what is its role?
+
+- Why we don't put the database as a deployment?
+
+###### Did the student reply correctly to the questions?
+
+##### Test The solution
+
+#### Inventory API Endpoints
+
+##### Open Postman and make a `POST` request to `http://[GATEWAY_IP]:[GATEWAY_PORT]/api/movies/` address with the following body as `Content-Type: application/json`:
+
+```json
+{
+  "title": "A new movie",
+  "description": "Very short description"
+}
+```
+
+###### Can you confirm the response was the success code `200`?
+
+##### In Postman make a `GET` request to `http://[GATEWAY_IP]:[GATEWAY_PORT]/api/movies/` address.
+
+###### Can you confirm the response was success code `200` and the body of the response is in `json` with the information of the last added movie?
+
+#### Billing API Endpoints
+
+##### Open Postman and make a `POST` request to `http://[GATEWAY_IP]:[GATEWAY_PORT]/api/billing/` address with the following body as `Content-Type: application/json`:
+
+```json
+{
+  "user_id": "20",
+  "number_of_items": "99",
+  "total_amount": "250"
+}
+```
+
+###### Can you confirm the response was success code `200`?
+
+##### Stop the billing-app container
+
+###### Can you confirm the `billing-app` container was correctly stopped?
+
+##### Open Postman and make a `POST` request to `http://[GATEWAY_IP]:[GATEWAY_PORT]/api/billing/` address with the following body as `Content-Type: application/json`:
+
+```json
+{
+  "user_id": "22",
+  "number_of_items": "10",
+  "total_amount": "50"
+}
+```
+
+###### Can you confirm the response was success code `200` even if the `billing_app` is not working?
+
+#### K8s components
+
+![suprise](../pictures/suprise.png)
+
+![components-of-kubernetes](../pictures/components-of-kubernetes.svg)
+
+In less than 15 minutes and with the help of google the student must explain all Kubernetes components and their roles.
+
+###### Can the learner explain the k8s components in less than 15 minutes?
+
+> If the learner is unable to explain the k8s component, this means that the learner has failed this audit, so he must re-read and spend more time on understanding!
```

Comment:
`components`.

... Also I would strongly suggest to rephrase or remove the last sentence as "he must re-read and spend more time on understanding!" seems a bit too aggressive for the public we are addressing and the general styling guidelines we use in the curriculum.

Reply:
component > components fixed

Reply:
I must insist the last sentence should change, it is way too aggressive and rude compared to our usual language style.

Reply:
done